frontend/app/components/date-from-to-edit.js

```diff
@@ -0,0 +1,61 @@
+import Component from '@ember/component';
+export default Component.extend({
+
+  init() {
+    this._super(...arguments);
+    this.months = Array(12).fill().map((x,i) => i + 1 + "");
+    if (!this.get('entity.isNew')) {
+      let date_from = this.get('entity.date_from');
+      let date_to = this.get('entity.date_to');
+      this.monthFrom = date_from.getMonth() + 1;
+      this.yearFrom = date_from.getFullYear();
+      this.monthTo = date_to.getMonth() + 1;
+      this.yearTo = date_to.getFullYear();
+    }
+
+  },
+
+  setMonth(month, option) {
+    let date = this.getDateObj(option)
+    date.setMonth(month - 1)
+    this.set('entity.date_' + option.toLowerCase(), date);
+    this.set('month' + option, month);
+  },
+
+  setYear(year, option) {
+    let date = this.getDateObj(option)
+    date.setFullYear(year)
+    this.set('entity.date_' + option.toLowerCase(), date);
+    this.set('year' + option, year);
+  },
+
+  getDateObj(option) {
```

Comment:
other name for method and param ?
maybe you could move this to the model class ? computed property